backend/middleware/upload.js

```diff
@@ -0,0 +1,12 @@
+const multer = require('multer')
+
+var storage = multer.diskStorage({
+    destination: (req, file, cb) => {
+        cb(null, 'uploads')
+    },
+    filename: (req, file, cb) => {
+        cb(null, file.fieldname + '-' + Date.now())
+    }
+});
+
+var upload = multer({storage: storage});
```

Comment:
Not sure if we need this middleware. I don't see it being referenced anywhere. Can we remove this upload.js file if we're not using it?

Reply:
Good point, I tested the code without the file and it doesn't impact the functionality of the project.